lib/PN5180/PN5180ISO14443.cpp

```
@@ -267,6 +267,11 @@ bool PN5180ISO14443::mifareHalt() {
 	return true;
 }
 
+bool PN5180ISO14443::mifareAuthenticate(uint8_t blockNo, uint8_t keyType, const uint8_t *key,
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)